.babelrc

```diff
@@ -3,7 +3,8 @@
     [
       "@babel/preset-env",
       {
-        "modules": false
+        "useBuiltIns": "usage",
+        "corejs": "3.23"
```

Comment:
to match the Webpack Encore Bundle recipe, see https://github.com/symfony/recipes/blob/main/symfony/webpack-encore-bundle/1.10/webpack.config.js#L55-L56